pkg/minikube/bootstrapper/bootstrapper.go

```diff
@@ -51,6 +51,7 @@ type Bootstrapper interface {
 const (
 	// BootstrapperTypeKubeadm is the kubeadm bootstrapper type
 	BootstrapperTypeKubeadm = "kubeadm"
+	BootstrapperTypeKICBS   = "kicbs"
```

Comment:
Does the BS at the end stand for bootstrapper? Just `BootstrapperTypeKic` might make more sense if that's the case!

Reply:
agreed :)